Lib/socket.py

```diff
@@ -463,6 +465,36 @@ def fromfd(fd, family, type, proto=0):
     nfd = dup(fd)
     return socket(family, type, proto, nfd)
 
+_GLOBAL_DEFAULT_MAXFDS = object()
+
+if hasattr(_socket.socket, "sendmsg"):
+    def send_fds(sock, buffers=None, fds=[], flags=[], address=None):
```

Comment:
Let's make ``buffers`` and ``fds`` required arguments and pass ``buffers`` directly to the ``sendmsg`` method.

``send_fds(sock, buffers, fds, flags=0, address=None):``